src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/AbstractRabbitMessageRouter.java

```diff
@@ -161,10 +171,19 @@ public void sendAll(T message, String... queueAttr) throws IOException {
      * @param filterStrategy filter strategy for filtering message fields
      * @throws NullPointerException if {@code filterStrategy} is null
      */
-    public void setFilterStrategy(FilterStrategy filterStrategy) {
+    public void setFilterStrategy(@NotNull FilterStrategy filterStrategy) {
```

Comment:
Maybe use the getDefaultFilterStrategy method insted

Reply:
Do you mean if the argument is null call `getDefaultFilterStrategy`?

Reply:
We have two methods to change the value of the filterStrategy variable:
* protected getDefaultFilterStrategy
* public setFilterStrategy
Maybe remove the setFilterStrategy method and will override the getDefaultFilterStrategy method in a child class